### ISSUE TITLE
Continuous auto-focus doesn't work on Android devices

### ISSUE THREAD

**Issue by bobtheowl** [opentok/cordova-plugin-opentok#76]
# Bug Report

**Current behavior**
> When performing a video chat on an Android device, the device camera doesn't auto-focus. When running on an iOS device, the auto-focus works as expected.

**Steps to reproduce**
> 1. Publish a video on an Ionic app using the Cordova plugin (we are using v3.1.2).
> 2. Make sure you are using the back camera.
> 3. Move the device to be very close and very far away from objects.

**What is the current *bug* behavior?**
> On Android devices, the focus remains at a fixed spot.

**What is the expected *correct* behavior?**
> On iOS devices, the focus adjusts as needed.

**Relevant logs and/or screenshots**
> [A community post on the TokBox website](https://support.tokbox.com/hc/en-us/community/posts/203945663-Is-it-possible-to-enable-continuous-autofocus-during-video-call-) references setting `Camera.Parameters.FOCUS_MODE_CONTINUOUS_VIDEO` in a custom video capturer class. I tried implementing this in the plugin but it didn't seem to have any effect.

**Comment by smuaq** [comment]
Is autofocus working at all on your side? Im having trouble having any focus at all on android devices!

**Comment by bobtheowl** [comment]
Yeah, the same app has functionality to record regular video using the native camera app and the autofocus works fine on that.

**Comment by smuaq** [comment]
I see that the config.xml has permission declared for auto focus: 
`<uses-feature android:name="android.hardware.camera.autofocus" android:required="false" />`
But its not working..

**Comment by msach22** [comment]
@bobtheowl @smuaq the autofocus require feature is set to `false`, does it work for you when you set it to `true`?

**Comment by smuaq** [comment]
No, I tried it with `true` but still the same
